PWGMM/Mult/Tasks/multiplicityPbPb.cxx

```diff
@@ -123,9 +130,14 @@ struct multiplicityPbPb {
 
       histos.fill(HIST("EtaZvtxTracks"), track.eta(), collision.posZ());
       histos.fill(HIST("PhiEtaTracks"), track.phi(), track.eta());
+
+      if(track.eta()<0.1) ++trackCounter_01;
```

Comment:
I think it is supposed to be `std::abs(track.eta()) < 0.1`?

Reply:
Yes, thank you so much!